src/rviz/TSRMarker.cpp

```diff
@@ -0,0 +1,14 @@
+#include <aikido/rviz/TSRMarker.hpp>
+
+using dart::dynamics::SimpleFrame;
+
+namespace aikido {
+namespace rviz {
+
+TSRMarker::TSRMarker(std::vector<std::unique_ptr<SimpleFrame>> tsrFrames)
```

Comment:
> Note that std::vector<std::unique_ptr<...>> is not copyable, so we already must not  be introducing an extra copy.

Hm, this is what I missed. It still prevents passing lvalue even without making it rvalue reference but in a different way (we get different error messages).

I still prefer to make the argument _explicitly_ rlvalue reference because we always expect to pass rlvalue here. However, I am not strongly against leaving it as it is since it works anyways as we intend.